packages/react-native-otlp/android/build.gradle

```diff
@@ -1,15 +1,15 @@
 buildscript {
   // Buildscript is evaluated before everything else so we can't use getExtOrDefault
   def kotlin_version = rootProject.ext.has("kotlinVersion") ? rootProject.ext.get("kotlinVersion") : project.properties["RNEmbraceOTLP_kotlinVersion"]
-  def gradle_version = project.properties["RNEmbraceOTLP_gradleVersion"]
 
   repositories {
     google()
     mavenCentral()
   }
 
   dependencies {
-    classpath "com.android.tools.build:gradle:$gradle_version"
+    classpath "com.android.tools.build:gradle:8.6.0"
```

Comment:
this is the one bump I'm still not fully sure on, looks like the create library tool outputs it as "7.2.1". Do you know if it has any impact on how the customer's app gets built? That might be hard to dig into to, alternatively could try keeping it as "7.2.1" and building on an app that has a higher version that's above our minimum required

Reply:
will take a look if I still need to run the `gradle wrapper` commando that would update what it is produced there

Reply:
done. I was able to set everything with a minimum of 7.2.1